### PR TITLE
Document a few undocumented things

### PR DESCRIPTION
**Pull request by lydell**

### CODE REVIEW

File: documentation/index.html.js [hunk 1]
```diff
@@ -862,8 +874,9 @@ Expressions
     <p>
       <span id="try" class="bookmark"></span>
       <b class="header">Try/Catch/Finally</b>
-      Try/catch statements are just about the same as JavaScript (although
-      they work as expressions).
+      Try/catch statements are just about the same as JavaScript, with a few
```
Comment (vendethiel) [inline documentation/index.html.js]: as in JavaScript/as JavaScript's (not sure if it's better)?

File: documentation/coffee/modulo.coffee [hunk 2]
```diff
@@ -0,0 +1,4 @@
+-7 % 5 == -2 # The reminder of 7 / 5
```
Comment (michaelficarra) [inline documentation/coffee/modulo.coffee]: remainder

**Comment by lydell** [comment]
Re-pushed.

**Comment by vendethiel** [comment]
:+1:

**Pull request merged**